### ISSUE TITLE
How to check the results of linear evaluation?

### ISSUE THREAD

**Issue by Hiiragi0107** [google-research/simclr#68]
I did a pre-training and linear evaluation in cifar10.
I would like to see the results of the linear evaluation.
How can I see the results? What kind of code do I need to type to see the results?
This is a rudimentary question, but thank you for your help.

**Comment by chentingpc** [comment]
please see the instruction here: https://github.com/google-research/simclr#finetuning-the-linear-head-linear-eval

**Comment by Hiiragi0107** [comment]
Thanks for the reply!
I typed that code and the following code to check the results.

python -m tensorboard.main --logdir=/tmp/simclr_test
python -m tensorboard.main --logdir=/tmp/simclr_test_ft

However, the only result I see as a graph is the train, not the validation graph.
Is it possible to see the validation graphs?
I'm sorry for my frequent visits.

**Comment by chentingpc** [comment]
there's no validation set here, and the current evaluation is one-time so the test result is just a point. you need to change the code in order to support validation set and continuous evaluation during training.

**Comment by Hiiragi0107** [comment]
I understand.
Thank you for your kind words.
